### PR TITLE
chore: publish v1.1.2

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

**Comment by Vinzent03** [comment]
@dshukertjr You know you have some merge conflicts?

**Comment by dshukertjr** [comment]
@Vinzent03 
Ah, how did that happen 😂 It should be good now!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+## [1.1.2]
+
+- fix: adds increment ID in channel topic inside stream() to enable listening to the same stream multiple times [(#161)](https://github.com/supabase/supabase-dart/pull/161)
```
Comment (Vinzent03) [inline CHANGELOG.md:4]: Is this relevant for the user or can we simplify it to allow multiple streams to same table? So to not mention the way we implemented it.

**Review by Vinzent03 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dshukertjr) [reply CHANGELOG.md:4]: @Vinzent03 Love the suggestion! Let me simplify it right now!

**Review by Vinzent03 — APPROVED** [review]
LGTM!

**Pull request merged**